### ISSUE TITLE
no separate paragraph, but rst lines separated by empty line

### ISSUE THREAD

**Issue by rpuntaie** [sphinx-doc/sphinx#4432]
This 

```
Example as shown in |table6|.

.. _`table6`:

|table6|: About this table.


.. |table6| replace:: :ref:`Table2.6<table6>`
```

results in one line instead of two paragraphs for latex output.

```
Example as shown in Table2.6. Table2.6: About this table.
```

**Comment by jfbu** [comment]
This shows in Sphinx 1.6.5 but is already fixed in 1.6.6. Can you try updating Sphinx?

**Comment by jfbu** [comment]
This was already fixed at https://github.com/sphinx-doc/sphinx/commit/29c62c64f5dfd989cca00028fff8936a324873b9 (#4208) for 1.6.6 release. Thanks for reporting!

duplicate of #4206
